platform/core/src/utils/isDisplaySetReconstructable.js

```diff
@@ -173,7 +224,12 @@ function _getPerpendicularDistance(a, b) {
 }
 
 const constructableModalities = ['MR', 'CT', 'PT', 'NM'];
-const reconstructionIssues = {
+export const reconstructionIssues = {
+  DATASET_4D: 'datasetis4D',
+  VARYING_IMAGESDIMENSIONS: 'imagesdimensionsvarying',
+  VARYING_IMAGESCOMPONENTS: 'imagescomponentsvarying',
```

Comment:
Lets extract these 2 enums to OHIF/core enums file if there is one or to create a new one, this way extensions can import these and use them for custom internal checks.

Reply:
ok moved them in OHIF/core/src/enums.js